README.md

```diff
@@ -1,4 +1,4 @@
-<!-- DO NOT EDIT. THIS IS GENERATED FROM README.md.gotmpl -->
+-e <!-- DO NOT EDIT. THIS IS GENERATED FROM README.md.gotmpl -->
```

Comment:
-e?

Reply:
straight up typo. Whooops

Reply:
@kylecarbs Apparently the `make fmt` placed that `-e`

Reply:
I'm not sure if enterprise-dev on arch behaves differently from Ubuntu (enterprise-dev-ubuntu), that might explain the difference here? The -e was part of the initial PR :man_shrugging: